base_app/src/app/app.component.ts

```diff
@@ -789,8 +789,10 @@ export class AppComponent implements OnInit, AfterViewInit /*, OnChanges */ {
                 remoteData$.subscribe(
                     res => {
                         if (res.status === 200 && typeof res.response === 'object' && (null !== res.response)) {
-                            tab._pendingRequestExpiration = 0; // Cancel wait
+                            tab._pendingRequestExpiration = 0; // Cancel wait                            
```

Comment:
should set up your IDE to trim trailing whitespace on save

Reply:
fixed whitespace issue and updated my vscode settings